clients/ui/frontend/src/app/pages/modelRegistry/screens/components/ArchiveRegisteredModelModal.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { Flex, FlexItem, Stack, StackItem, TextInput } from '@patternfly/react-core';
+import { Modal } from '@patternfly/react-core/deprecated';
```

Comment:
I'm also hitting this in my current PR that's not up yet. Will we want to update all of our Modals in a future sprint?